.github/workflows/create-draft-release.yml

```diff
@@ -0,0 +1,63 @@
+name: "Create Draft Release"
+
+env:
+  BUILDDIR: "build"
+  DEVELOPER_DIR: "/Applications/Xcode_12.5.1.app/Contents/Developer"
+
+on:
+  workflow_dispatch
+
+concurrency: 
+  group: publish-release-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: "Publish binaries for release"
+    runs-on: macos-11
+
+    steps:
+      - name: "Checkout sources"
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: "Build release distribution"
+        run: make release
+        env:
+          GITHUB_ACTOR: ${{ github.actor }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: "Extract version information from project"
+        run: |
+          MARKETING_VERSION=$(xcrun xcodebuild -showBuildSettings -project Sparkle.xcodeproj | grep MARKETING_VERSION | cut -f 7 -d ' ')
```

Comment:
Can you enlighten me on the understandings of this unix cut incantation, specifically the `-f 7` part.

Reply:
Right, so my understanding here is that the invocation of cut is targeting only spaces. So the value here is after the 7th space in the string. 

It's not great, but it's super fast. I should probably use a regex here so that it's a bit more reliable.

Reply:
Done in b2e07a51f8fdd769da8aaded3177968587f4de7d.